Publisher/nl/productupdate20230511.md

```diff
@@ -0,0 +1,35 @@
+# Productupdate - Opvolgactie verbeteringen, uitschrijfbevestiging en verzendmomenten uitsluiten met RRule
+
+## Opvolgactie op basis van veldwijziging
+Het is nu mogelijk om in het _bestemming checken_ blok te kijken of een veld is aangepast. Hierbij kan je aangeven dat de opvolgactie uitgevoerd moet worden als het veld naar een specifieke waarde is aangepast of bij iedere wijziging waarbij de waarde niet gespecificeerd is. 
```

Comment:
We hebben een nieuwe functie toegevoegd waarmee je een opvolgactie kunt uitvoeren op basis van wijzigingen in een veld. Met het _bestemming checken_ blok kun je nu controleren of een veld is aangepast. Je kunt specificeren dat de opvolgactie moet worden uitgevoerd als het veld naar een specifieke waarde is aangepast of bij elke wijziging waarbij de waarde niet is gespecificeerd.